packages/app/src/runner/ResizablePanels.spec.tsx

```diff
@@ -0,0 +1,185 @@
+import ResizablePanels, { ResizablePanelName, DraggablePanel } from './ResizablePanels.vue'
+
+// default values
+const defaultPanel1Width = 280
```

Comment:
We might want to name these something more descriptive like SpecPanel and ReporterPanel, I feel like we can do better than numbers 🤔